config.json

```diff
@@ -152,6 +152,7 @@
           "sequences"
         ]
       },
+
```

Comment:
Was this empty line added on purpose?

Reply:
Probably an artifact from a bunch of rebasing and splitting these things up into nice bite size chunks.